### PR TITLE
T-32185 Migrate Ellipsis Finance

### PR DESCRIPTION
**Pull request by antonio-mendes**
Migrating Ellipsis Finance to DuneSQL.

Marking as WIP since I'm having issues with fields from a decoded table and need to dig further.

### CODE REVIEW

**Comment by antonio-mendes** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by antonio-mendes** [comment]
recheck

**Comment by jeff-dude** [comment]
rebasing with main helped get past the tag issue (those models were merged last night)

**Comment by antonio-mendes** [comment]
@jeff-dude this issue is no longer showing up on my personal env even after forcing a full-refresh for the `ellipsis_finance_bnb_pool` model. Let's maybe sync later and try to figure it out

**Comment by jeff-dude** [comment]
> @antonio-mendes looks like there are some failing tests here

curve is hitting similar failures. just needs some seed/generic test fixing on the pools. let's ensure only one of us goes ahead to fix, then can apply on the other

File: models/ellipsis_finance/bnb/ellipsis_finance_bnb_pool_tokens.sql [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {{ config(
+    tags=['dunesql'],
     alias = alias('pool_tokens'),
-    partition_by = ['pool'],
```
Comment (aalan3) [inline models/ellipsis_finance/bnb/ellipsis_finance_bnb_pool_tokens.sql:3]: Should partition by `block_month`

File: models/ellipsis_finance/ellipsis_finance_trades.sql
Comment (jeff-dude) [inline models/ellipsis_finance/ellipsis_finance_trades.sql:1]: you should add `block_month` in this spell, then also add `ellipsis_finance_trades` to `dex_trades_migration_beta` -- it already exists there, just sitting in comments instead, we can move to actual code and see it run successfully here 🤝

**Review by jeff-dude — CHANGES_REQUESTED** [review]
lfg. one minor note below

**Comment by antonio-mendes** [comment]
I think we might be good now @jeff-dude ?

**Comment by jeff-dude** [comment]
> I think we might be good now @jeff-dude ?

yep, let's just let CI ride to be sure

**Review by jeff-dude — APPROVED** [review]

**Pull request merged**